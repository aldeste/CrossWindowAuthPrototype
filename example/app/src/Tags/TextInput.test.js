import TextInput from "./TextInput";
import renderer from "react-test-renderer";
import React from "react";
import "jest-styled-components";

describe("TextInput tag", () => {
  it("Should be defined", () => {
    expect(TextInput).toBeDefined();
  });

  it("Is a styled component", () => {
    expect(TextInput.name).toBe("StyledComponent");
  });

  it("Matches earlier configuration", () => {
    const tree = renderer.create(<TextInput />).toJSON();
    expect(tree).toMatchStyledComponentsSnapshot();
  });
});
