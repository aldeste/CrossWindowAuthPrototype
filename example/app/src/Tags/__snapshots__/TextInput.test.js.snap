// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput tag Matches earlier configuration 1`] = `
.daQrd {
  font-family: inherit;
  font-weight: inherit;
  font-size: inherit;
  line-height: inherit;
  padding: 0.5rem 1rem;
  cursor: text;
  color: inherit;
  border: solid 1px #333;
  background-color: transparent;
}

<input
  className="sc-bdVaJa daQrd"
/>
`;
